### ISSUE TITLE
Teach Rich Remix 👁

### ISSUE THREAD

**Issue by RichStone** [bouwe77/curious-coders#23]
Revenge for #3

**Comment by bouwe77** [comment]
https://media.giphy.com/media/CDJo4EgHwbaPS/giphy.gif

**Comment by bouwe77** [comment]
![](https://media.giphy.com/media/CDJo4EgHwbaPS/giphy.gif)

**Comment by DavidMatalik** [comment]
I would be very interested in this topic

**Comment by DavidMatalik** [comment]
Rename it into teach Rich + David Remix

**Comment by RichStone** [comment]
yeah man, the other day I heard about it on the Remote Ruby podcast, people following it from the depth of the Ruby and Rails universe. THIS MUST BE IMPORTANTE

**Comment by bouwe77** [comment]
It seems Remix has its Rails influences, but I don't know them. But I guess that's where you come in then... 😏
